### ISSUE TITLE
Boost version problems

### ISSUE THREAD

**Issue by percolator** [statisticalbiotechnology/maracluster#24]
I get errors when compiling maracluster on my ubuntu 20.04 installation. Do you understand what is wrong?

$ git clone git@github.com:statisticalbiotechnology/maracluster.git; cd maracluster; bash quickbuild.sh
[.....]
- Architecture: amd64
-- Checking packages:
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake:117 (find_package):
  Found package configuration file:

    /usr/lib/x86_64-linux-gnu/cmake/boost_filesystem-1.71.0/boost_filesystem-config.cmake

  but it set boost_filesystem_FOUND to FALSE so package "boost_filesystem" is
  considered to be NOT FOUND.  Reason given by package:

  No suitable build variant has been found.

  The following variants have been tried and rejected:

  * libboost_filesystem.so.1.71.0 (shared, Boost_USE_STATIC_LIBS=ON)

  * libboost_filesystem.a (shared runtime, Boost_USE_STATIC_RUNTIME=ON)

Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake:182 (boost_find_component)
  /usr/share/cmake-3.16/Modules/FindBoost.cmake:443 (find_package)
  src/CMakeLists.txt:22 (find_package)

-- Configuring incomplete, errors occurred!
See also "/tmp/tst/build/ubuntu64/maracluster/CMakeFiles/CMakeOutput.log".

**Comment by MatthewThe** [comment]
Not sure what the CMake errors mean, but it should be using the Boost version from ProteoWizard (v1.76.0) at `/tmp/tst/build/ubuntu64/tools/` and not v1.71.0 from `/usr/lib/...`.

Do you still have the full build log? Perhaps the ProteoWizard build failed.

**Comment by percolator** [comment]
Yes, it seems to be the install_proteowizard.sh scripts that fails to get
the right download. Could proteowizard have changed their naming
conventions?

--Lukas

**Comment by percolator** [comment]
I forgot to attach error logs.
$ cat pwiz_installation.log
/tmp/maracluster/..//maracluster/admin/builders/install_proteowizard.sh:
line 30: ./quickbuild.sh: No such file or directory

and the maracluster quick_build log is attached
